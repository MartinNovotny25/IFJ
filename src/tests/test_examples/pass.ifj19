def main():
    def main1():
        def main2():
            def main3():
                def main4():
                    def main5():
                        pass


if 1<2:
    if 1<2:
        if 1<2:
            if 1<2:
                if 1<2:
                    if 1<2:
                        pass
