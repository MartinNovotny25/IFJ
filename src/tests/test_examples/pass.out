TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(IDENTIFIKATOR)
TOKEN(()
TOKEN())
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(INTEGER, 1)
TOKEN(<)
TOKEN(INTEGER, 2)
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(INTEGER, 1)
TOKEN(<)
TOKEN(INTEGER, 2)
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(INTEGER, 1)
TOKEN(<)
TOKEN(INTEGER, 2)
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(INTEGER, 1)
TOKEN(<)
TOKEN(INTEGER, 2)
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(INTEGER, 1)
TOKEN(<)
TOKEN(INTEGER, 2)
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(INTEGER, 1)
TOKEN(<)
TOKEN(INTEGER, 2)
TOKEN(:)
TOKEN(EOL)
TOKEN(INDENT)
TOKEN(KLICOVE_SLOVO)
TOKEN(EOL)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(DEDENT)
TOKEN(EOF)
